Apps/phokta/okta_view.py

```diff
@@ -12,15 +12,21 @@ def get_ctx_result(result):
     param = result.get_param()
     summary = result.get_summary()
     data = result.get_data()
+    message = result.get_message()
+    status = result.get_status()
 
     ctx_result['param'] = param
+    ctx_result['status'] = status
 
     if (data):
         ctx_result['data'] = data
 
     if (summary):
         ctx_result['summary'] = summary
 
+    if (message):
```

Comment:
can we remove the parentheses around  these if conditions?

Reply:
Fixed